src/config.rs

```diff
@@ -230,10 +308,8 @@ fn search_snare_conf() -> Option<PathBuf> {
     None
 }
 
-pub enum GenericOption<StorageT> {
-    Email(Lexeme<StorageT>),
-    MaxJobs(Lexeme<StorageT>),
-    Port(Lexeme<StorageT>),
-    ReposDir(Lexeme<StorageT>),
-    Secret(Lexeme<StorageT>),
+/// The configuration for a given repository.
+pub struct RConfig<'a> {
+    pub email: Option<&'a str>,
```

Comment:
Who owns the data behind these references? The parser/lexer?

Reply:
The config.